Assets/AppCenter/Plugins/AppCenterSDK/Core/UWP/AppCenterInternal.cs

```diff
@@ -200,26 +205,17 @@ private static void Prepare()
                     UnityScreenSizeProvider.Initialize();
                     DeviceInformationHelper.SetScreenSizeProviderFactory(new UnityScreenSizeProviderFactory());
 
-#if ENABLE_IL2CPP
-#pragma warning disable 612
                     /**
                      * Workaround for known IL2CPP issue.
                      * See https://issuetracker.unity3d.com/issues/il2cpp-use-of-windows-dot-foundation-dot-collections-dot-propertyset-throws-a-notsupportedexception-on-uwp
                      *
                      * NotSupportedException: Cannot call method
                      * 'System.Boolean System.Runtime.InteropServices.WindowsRuntime.IMapToIDictionaryAdapter`2::System.Collections.Generic.IDictionary`2.TryGetValue(TKey,TValue&)'.
                      * IL2CPP does not yet support calling this projected method.
+                     * TODO: Can be removed after migration for settings is added.
```

Comment:
What is that?

Reply:
This workaround is not required anymore but it uses a different place to store settings (enabling state etc). So to remove it we should provide some migration. So we can just keep it for now

Reply:
Clarified that in comment